<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeDesk</class>
 <widget class="QMainWindow" name="EmployeeDesk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EmployeeDesk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>EmployeeDesk</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="viewEmployeesBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Список
сотрудников</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resControlBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Контроль
ресурсов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewTasksBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Просмотр
задач</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewProfileBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Профиль</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
